docker/test_env/non_dev_geth.go

```diff
@@ -1,426 +0,0 @@
-package test_env
-
-import (
-	"encoding/hex"
-	"fmt"
-	"io"
-	"os"
-	"strconv"
-	"strings"
-	"testing"
-	"time"
-
-	"github.com/docker/docker/api/types/container"
-	"github.com/docker/docker/api/types/mount"
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/ethclient"
-	"github.com/google/uuid"
-	"github.com/rs/zerolog"
-	"github.com/rs/zerolog/log"
-	tc "github.com/testcontainers/testcontainers-go"
-	tcwait "github.com/testcontainers/testcontainers-go/wait"
-
-	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
-	"github.com/smartcontractkit/chainlink-testing-framework/docker"
-	"github.com/smartcontractkit/chainlink-testing-framework/logging"
-	"github.com/smartcontractkit/chainlink-testing-framework/mirror"
-	"github.com/smartcontractkit/chainlink-testing-framework/utils/templates"
-	"github.com/smartcontractkit/chainlink-testing-framework/utils/testcontext"
-)
-
-const (
-	TX_NON_DEV_GETH_WS_PORT = "8546"
-	BOOTNODE_PORT           = "30301"
-
-	defaultNonDevGethImage         = "ethereum/client-go:v1.12.0"
-	defaultNonDevGethAllToolsImage = "ethereum/client-go:alltools-v1.10.25"
```

Comment:
Is anything else using this image? If not we should remove it from the mirror workflow.

Reply:
afaik nobody else is using it, we can remove them